src/randoop/sequence/SequenceExceptionError.java

```diff
@@ -0,0 +1,29 @@
+package randoop.sequence;
+
+public class SequenceExceptionError extends Error {
+
+  /** ID for serialization */
+  private static final long serialVersionUID = 4778297090156993454L;
+  private Throwable e;
+  private ExecutableSequence sequence;
+  private int pos;
```

Comment:
Please document the fields.  I am not sure what "pos" represents.  Likewise, document in the constructor.